src/main/resources/db/migration/V1__Create_db.sql

```diff
@@ -0,0 +1,45 @@
+create table DRIVER
```

Comment:
Do u implement the connection between tables via foreign key and primary key

Reply:
done

Reply:
I can see find any changes, could u show me?